### DISCUSSION TITLE
v1.26 feedback thread

### DISCUSSION THREAD

**Discussion by Lailloken** [Lailloken/Exile-UI#72] [Ideas & Feedback]
Highlighted changes:
- added a timeless jewel seed explorer

**Comment by redrios** [comment]
im silly I can't find the explorer :( don't know how to trigger it
lemme re-install

**Reply by Lailloken** [reply]
You can access it via the context menu (clicking middle mouse-button on a timeless jewel) or by typing .legion into chat.

**Reply by redrios** [reply]
thank you!
